src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -77,12 +57,12 @@ object JesBackend {
   }
 
   def workflowGcsPath(descriptor: WorkflowDescriptor): String = {
-    val bucket = descriptor.workflowOptions.getOrElse(GcsRootOptionKey, JesConf.getString("baseExecutionBucket"))
+    val bucket = descriptor.workflowOptions.getOrElse(GcsRootOptionKey, JesConf.executionBucket)
```

Comment:
@cjllanwarne I moved the conf related methods to the class so the whole thing can be mocked, we just need to be aware that the side effect is that there is now a new instance of `JesConf` and `JesConnection` for every `JesBackend` instance that we spin, which I don't expect to be more than one I think but anyway.